wp-content/themes/eighshift-task/src/blocks/custom/image-text/components/image-text-options.js

```diff
@@ -0,0 +1,35 @@
+import { __ } from '@wordpress/i18n';
+import { PanelBody, SelectControl, CheckboxControl } from '@wordpress/components';
+
+export const ImageTextOptions = (props) => {
+  const {
+    mediaPosition,
+    onChangeMediaPosition,
+    includeBtn,
+    onChangeIncludeBtn,
+  } = props;
+
+  return (
+    <PanelBody title={__('Image Text Settings', 'eighshift-task')}>
+      {mediaPosition &&
+        <SelectControl
+          label={__('Image Position', 'eighshift-task')}
+          value={mediaPosition}
+          options={[
+            { label: __('Left', 'eighshift-task'), value: 'left' },
+            { label: __('Right', 'eighshift-task'), value: 'right' },
+          ]}
+          onChange={onChangeMediaPosition}
+        />
+      }
+
+      {onChangeIncludeBtn &&
+        <CheckboxControl
```

Comment:
try implementing https://github.com/WordPress/gutenberg/tree/master/packages/components/src/toggle-control it looks better and is more UX friendly